Project/src/MakeCall/CallCard.js

```diff
@@ -593,16 +592,10 @@ export default class CallCard extends React.Component {
                     <div className="ms-Grid-col ms-lg6">
                         <h2>{this.state.callState !== 'Connected' ? `${this.state.callState}...` : `Connected`}</h2>
                     </div>
-                    <div className="ms-Grid-col ms-lg6 text-right">
-                        {
-                            this.call &&
-                            <h2>Call Id: {this.state.callId}</h2>
-                        }
-                        {
-                            this.call &&
-                            <h2>Sent Resolution: {this.state.sentResolution}</h2>
-                        }
-                    </div>
+                    {
+                        this.call &&
```

Comment:
right here, check if the feature exists and only then render the component to proof that feature exists before rendering the component at all

Reply:
Explained the logic internally